src/renderer/services/client.js

```diff
@@ -669,9 +671,10 @@ export default class ClientService {
    * @returns {Object}
    */
   async buildTransfer ({ amount, fee, recipientId, vendorField, passphrase, secondPassphrase, wif }, isAdvancedFee = false, returnObject = false) {
+    const staticFee = store.getters['transaction/staticFee'](0) || V1.fees[0]
     // To ensure that transfers cannot be build with a bigger fee than V1
```

Comment:
```suggestion
    // To ensure that transfers cannot be built with bigger fees than the static fees allowed by the peer or V1
```

or remove the comment if you think that is unnecessary

Reply:
I will remove as it's only in the transfer method, thanks 👌